map_model/src/objects/road.rs

```diff
@@ -269,6 +269,12 @@ impl Road {
             .min_by_key(|(idx, _)| (our_idx - (*idx as isize)).abs())
             .map(|(_, l)| l)
     }
+    /// This function returns the lane closest to the center from the left hand side of the street
+    pub fn get_lane_underneath_label(&self) -> LaneType {
+        let size = self.lanes.len() - 1;
+        let lane_type_under_label: LaneType = self.lanes[size / 2].lane_type;
```

Comment:
To match the name of the method, we could return the lane itself, and the caller could then lookup the lane type. We might have some future need to do something similar! The return type would be `&Lane` (a borrow of a Lane), and you could just do `&self.lanes[size / 2]`